### PR TITLE
Docker build of Apollo-11 sources

### PR DESCRIPTION
**Pull request by jimlawton**
This is a first attempt at a Docker configuration for a build of the Apollo-11 sources using the VirtualAgc tools.

To use it, you'll need Docker (on Linux, Mac or Windows). Then do:
`  $ docker build -t apollo-11 .`

It first clones and builds the VirtualAgc tools and mission code. Then it compiles the Apollo-11 sources (from the current directory) and compares the generated rope binaries against those in the VirtualAgc build.

This is based on an Ubuntu 16.04 base image.

Note that the VirtualAgc tools are sensitive to spaces/tabs, so assembly errors may result from using a different formatting standard.

### CODE REVIEW

**Comment by jimlawton** [comment]
You folks may not be interested in this, but if anyone wants to try assembling the Apollo-11 source code, this is a relatively simple way - no installing a lot of dependencies.

**Comment by oldmud0** [comment]
This actually looks quite interesting, but sadly I cannot add it to the repository because it is meant to be for preservation (i.e. no build tools, just the code that Apollo 11 launched with). Perhaps you can make a repository of your own and even hook a continuous integration service to it and the whole shebang. That would be cool :smiley:

**Comment by jimlawton** [comment]
No worries. I've already done that for the `virtualagc` repo.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+RUN apt-get -y install make
+RUN apt-get -y install g++
+RUN apt-get -y install python
+RUN apt-get -y install libncurses5
+RUN apt-get -y install libncurses5-dev
+
+RUN git clone https://github.com/rburkey2005/virtualagc
+
+# Use this to build a copy of the current directory. 
+RUN mkdir /apollo-11
```
Comment (MaximeHeckel) [inline Dockerfile]: You can reduce the number of layers here by using a single RUN command

**Pull request closed without merging**